app/components/avo/fields/file_field/edit_component.html.erb

```diff
@@ -1,20 +1,33 @@
 <%= field_wrapper **field_wrapper_args do %>
-  <% if @field.value.present? %>
+  <%= content_tag :div, data: {
+    controller: "file-field",
+  } do %>
     <div class="mb-2">
```

Comment:
Can we add the `mb-2` class inside that parent `div` so we won't have two nested divs?